packages/types/package.json

```diff
@@ -62,9 +62,10 @@
     "lint": "biome check src/ test/",
     "lint:fix": "yarn run lint --write",
     "test": "yarn test:unit",
-    "test:constants:minimal": "vitest run --project constants",
+    "test:constants:minimal": "vitest run --project constants-minimal",
     "test:constants:mainnet": "vitest run --project constants-mainnet",
-    "test:unit": "wrapper() { yarn test:constants:minimal $@ && yarn test:constants:mainnet $@ && LODESTAR_PRESET=mainnet vitest run --project unit $@; }; wrapper",
+    "test:constants": "vitest run --project constants-minimal --project constants-mainnet",
+    "test:unit": "vitest run --project unit-minimal --project unit-mainnet",
```

Comment:
do we want a command that combines `test:constants` and `test:unit`, decided to keep them separate to just have a single command in each yarn script, also gets rid of `wrapper()`

Reply:
separate constants tests have been removed and are now part of unit tests